src/pages/pto/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { NextPage } from "next";
+import { useQuery } from "react-query";
+import { useRouter } from "next/router";
+
+import { PTO, getPTOData } from "../../store";
+import Box from "../../components/Box";
+import GridContainer from "../../components/GridContainer";
+import Loading from "../../components/Loading";
+import Button from "../../components/Button";
+
+import styles from "./styles.module.scss";
+
+const Dashboard: NextPage = () => {
+  const { push } = useRouter();
+  const { data, isLoading, error } = useQuery<PTO>("pto", getPTOData, {
+    retry: 1,
+    staleTime: 1000 * 60 * 60, // 1 hour
+    cacheTime: 1000 * 60 * 10, // 10 minutes
+  });
+
+  if (isLoading) {
+    return <Loading />;
+  }
+
+  if (error) {
+    return <div>Something went wrong</div>;
+  }
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.logout}>
+        <Button onClick={() => push("/")}>Logout</Button>
```

Comment:
Does this actually have a logout functionality? If not could it be commented?

I think that the next improvement (for another PR) could be adding functionality to the useLogout hook.

Reply:
Navigating to home simulated logging out for now. Could add functionality to clear the token from session storage as well.

Reply:
I'm not familiar with what the OAuth approach should do in this case, maybe cleaning the session is not enough from a backend perspective...

Reply:
The login action gives us the code in the url and the token endpoint gives us the access token. No other with info is given to the browser from what I see.